### PR TITLE
[UPDATE] Updated 'About Docs' and added 'Navbar' page in Docs Section

### PR DESCRIPTION
**Pull request by himanshu-03**
- Updated  `About Docs` page in Docs Section
- Changed the label of `Docs Site` to `Docs` in `sidebar.js` file

### CODE REVIEW

**Review by vish-rt — APPROVED** [review]
Looks good to me

File: sidebars.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const sidebars =
       [
         {
           type: 'category',
-          label: 'Docs Site',
+          label: 'Docs',
```
Comment (demonkillerr) [inline sidebars.js:40]: We need to specify what this documentation is about. Calling it **'docs'** is very counter-intuitive.

**Review by demonkillerr — CHANGES_REQUESTED** [review]
Rename **Docs** to something else, or keep it that way

File: sidebars.js (hunk 1 above)
Comment (himanshu-03) [reply sidebars.js]: Changes updated!

**Comment by himanshu-03** [comment]
- Updated  `About Docs` page in Docs Section
- Restored the label of `Docs` to `Docs Site` in `sidebar.js` file
- Added and updated 'Navbar' page in Docs Section

**Review by vish-rt — APPROVED** [review]

**Review by demonkillerr — APPROVED** [review]
looks good

**Pull request merged**